core/templates/pages/library-page/search-selectors/subject-selector-modal.component.html

```diff
@@ -0,0 +1,2 @@
+<div>
```

Comment:
Please help me to understand, why empty divs?

Reply:
Ah they are there just as a placeholder, in a future PR we will add the proper html for this component. An empty html file would have worked too I think. This is being done like this because we were asked to split a large PR into smaller ones, to keep the lines per PR low.

Reply:
Sorry, I am not getting the complete picture of what HTML you want to put here in the future. Can you please add a comment here?
Also, is there any issue that tracks and explains how the future PR will look like?

Reply:
I removed the empty divs and added a comment. Also, there is an [open PR](https://github.com/oppia/oppia/pull/20432) which we were asked to split so there wouldn't be so many lines in a review, you can see how it will look like even though we'll make some changes, here's the [link](https://github.com/oppia/oppia/pull/20432/commits/5bf01e993605cbca329e363ab7c3bd5cb933dfcd#diff-3761eca5d20ed6565a4d74a7b1835a4b2a7c0ababf51509d1188fd85875b5a1a) to the html of this component.

Reply:
Generally, we try to make PR atomic so why can't we add HTML changes here itself?
If a component is expected to be large then it should be large enough, we can't create 3 different PRs for a single component.

Can we update this PR to make it atomic?
Also, the referred PR link in the above link is closed.
Thank you

Reply:
Done